.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{cpp,h,py}]
+charset = utf-8
```

Comment:
Is there a file which shouldn't be utf-8?

If there is, at least add `.md` here, otherwise just make this generic.